src/cmd/Makefile.am

```diff
@@ -2,13 +2,16 @@ AM_CXXFLAGS = \
     $(WARNING_CXXFLAGS) \
     $(CODE_COVERAGE_CXXFLAGS)
 
-AM_LDFLAGS = $(CODE_COVERAGE_LDFLAGS)
+AM_LDFLAGS = $(CODE_COVERAGE_LDFLAGS) $(SQLITE_LIBS)
 
-AM_CPPFLAGS = $(FLUX_CORE_CFLAGS)
+AM_CPPFLAGS = -I$(top_srcdir) $(FLUX_CORE_CFLAGS)
```

Comment:
Commit message:

 * typo: `Soltuion` :arrow_right: `Solution`
 * Suggestion: use C++ "source files" or "sources" not "scripts"

Reply:
Oops 🤦‍♂️ thanks for catching this. I just pushed up a fix to address the typos and wrong word choice for the commit message.